mead/config/conll-no-crf.json

```diff
@@ -54,7 +54,7 @@
     "dropout": 0.5,
     "rnntype": "blstm",
     "layers": 1,
-    "constrain_decode": true,
+    "constrain_decode": true, 
```

Comment:
Extra whitespace added?

Reply:
was wondering if you would notice that ;)